integration/setup.go

```diff
@@ -38,9 +39,10 @@ import (
 )
 
 var (
-	portF      = flag.Int("port", 0, "port to use; if 0, in-process FerretDB is used")
-	handlerF   = flag.String("handler", "pg", "handler to use for in-process FerretDB")
-	proxyAddrF = flag.String("proxy-addr", "", "proxy to use for in-process FerretDB")
+	portF       = flag.Int("port", 0, "first system's port for tests; if 0, in-process FerretDB is used")
+	proxyAddrF  = flag.String("proxy-addr", "", "proxy to use for in-process FerretDB")
+	handlerF    = flag.String("handler", "pg", "handler to use for in-process FerretDB")
+	compatPortF = flag.Int("compat-port", 37017, "second system's port for compatibility tests; if 0, they are skipped")
```

Comment:
And this one is something like "reference system" I guess? Or is it the other way around?

Reply:
Well, I would not call it a reference system because that's FerretDB when the target system (a name I like) is MongoDB